easybuild/easyconfigs/l/libStatGen/libStatGen-1.0.14-zlib.patch

```diff
@@ -0,0 +1,37 @@
+diff -ru libStatGen-1.0.14.orig/general/Makefile libStatGen-1.0.14/general/Makefile
```

Comment:
@boegel I had issues with the `make test` step before applying the patch. I think this is because I have zlib-dev rpm installed in my system